client/src/utils/mutations.js

```diff
@@ -174,4 +174,10 @@ export const ORDER_HISTORY = gql`
             }
         }
     }
+`;
+
+export const SEND_EMAIL = gql`
+    mutation sendEmail($username: String!, $email: String!, $message: String!) {
+        sendEmail(username: $username, email: $email, message: $message)
```

Comment:
Since the user is does not have to be logged in to contact us, I don't think the username field should be required.